pkg/prometheus/promcfg_test.go

```diff
@@ -5483,7 +5483,7 @@ func TestScrapeConfigSpecConfig(t *testing.T) {
 					},
 				},
 			},
-			golden: "ScrapeConfigSpecConfig_ProxySettingsIncompatiblePrometheusVersion.golden",
+			golden: "ScrapeConfigSpecConfig_ProxySettingsExceptProxyURLIncompatiblePrometheusVersion.golden",
```

Comment:
I've noticed that you renamed the golden file being used here, but there are no files with changed names in this PR. Should we delete the old golden file? (ScrapeConfigSpecConfig_ProxySettingsIncompatiblePrometheusVersion)